video_prediction/lstm_ops.py

```diff
@@ -99,7 +97,9 @@ def basic_conv_lstm_cell(inputs,
                             scope='Gates')
 
     # i = input_gate, j = new_input, f = forget_gate, o = output_gate
-    i, j, f, o = tf.split(axis=3, num_or_size_splits=4, value=i_j_f_o)
+
+    i, j, f, o = tf.split(i_j_f_o, 4,3 )
```

Comment:
Yes,you're right, seems my misunderstanding comes from the code from video_prediction/prediction_train.py line116: actions = tf.split(axis=1, num_or_size_splits=actions.get_shape()[1], value=actions), the IndexError:list index out of range shows because of the num_or_size_splits should be an int not a tf.dimension variable.